tests/unit/moduleapi/keyspace_events.tcl

```diff
@@ -87,6 +87,25 @@ tags "modules" {
             $rd1 close
         }
 
+        test "Keyspace notifications: unsubscribe removes handler" {
+            r config set notify-keyspace-events Kd
+            r del x
+            set rd1 [redis_deferring_client]
+            assert_equal {1} [psubscribe $rd1 *]
+            r keyspace.notify x
+            assert_equal {pmessage * __keyspace@9__:x notify} [$rd1 read]
+
+            r keyspace.unsubscribe 4 ;# 4 = REDISMODULE_NOTIFY_GENERIC
```

Comment:
IIRC, we don't use `g`(generic commands)  in the `notify-keyspace-events`, can we get the callback?